src/CLI/OptionsEvaluator.cs

```diff
@@ -56,5 +55,15 @@ public static IOptions ProcessInputs(Options rawInputs, IProcessHelper processHe
                 VerbosityLevel = verbosityLevel,
             };
         }
+
+        private static string GetReducedProcessName(string processName)
```

Comment:
Since "Reduced" means remove exe, "TrimDotExe" might be a better name for the function.